06_puppy/sirigithub/stores_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storesSuite struct {
+	suite.Suite
+	store     Storer
+	newStorer func() Storer
+	puppy1    Puppy
+}
+
+func (suite *storesSuite) SetupTest() {
+	suite.store = suite.newStorer()
+	suite.puppy1 = Puppy{Breed: "cavalier", Color: "White", Value: "$1700"}
+	suite.store.CreatePuppy(&suite.puppy1)
+}
+func (suite *storesSuite) TestCreatePuppySuccesful() {
+	//given
+	assert := assert.New(suite.T())
+	newPuppy := Puppy{Breed: "Dobberman", Color: "Black", Value: "$800"}
+	//when
+	id := suite.store.CreatePuppy(&newPuppy)
+	//then
+	assert.True(newPuppy.ID == suite.puppy1.ID+1, "Puppy Id increments sequentially")
+	puppy, err := suite.store.ReadPuppy(id)
+	if assert.NoError(err, "Read of created puppy should be successful") {
+		assert.Equal(puppy, &newPuppy)
+	}
+}
+func (suite *storesSuite) TestReadPuppySuccesful() {
+	//given
```

Comment:
definitely remove this `//given` comment - there is nothing actually "given" here, you're just aliasing `assert` to make the rest simpler (i.e. not really part of what is actually being tested).